### ISSUE TITLE
Web: Aww snap

### ISSUE THREAD

**Issue by FredrikNoren** [AmbientRun/Ambient#932]
After running for a while chrome crashes sometimes

**Comment by Pombal** [comment]
@FredrikNoren Do you have any more info on this? What were you running, for how long, what were you doing, etc.
I recently did a soak test with the skinmesh example and didn't see any crashes

**Comment by philpax** [comment]
I'm getting this pretty consistently; I think Ambient's not being unloaded properly when you switch pages, so it runs and eventually takes the browser down, maybe from trying to connect after disconnecting or something

**Comment by FredrikNoren** [comment]
Yeah same here. Let's move it back to 0.3 and see if we can do anything about it
